test/uk/gov/hmrc/eeittadminfrontend/ApplicationComponentsOnePerSuite.scala

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2018 HM Revenue & Customs
+ * Copyright 2019 HM Revenue & Customs
```

Comment:
As I said yesterday, you ought to have a look at the "Files changed" in GitHub before making creating the pull request, then you would see this.  As it happens, I am now not too worried about this, because now all of the date changes are in test files after the main files (& there are no other test changes).

I'm surprised though, because changing the main files but not the test files is a common mistake that comes from having done "sbt compile" or "sbt run", but not having done "sbt test".  That is why I suggested doing "sbt test"